src/nodes.coffee

```diff
@@ -3444,6 +3444,11 @@ exports.Expansion = class Expansion extends Base
 
   eachName: (iterator) ->
 
+  astType: -> 'RestElement'
```

Comment:
While `Expansion`s are Coffeescript-specific, it makes sense to model them like their close cousins splats, so `type: 'RestElement'` just with a `null` `argument` field

Reply:
Unless `RestElement` with a `null` `argument` might someday be used for something else in the Babel spec?

Reply:
That's an interesting consideration. Conceptually I think this representation makes sense (an expansion basically is a "spread without an argument") and I'd tend not to worry too hard about potential future evolution of the Babel spec (as we've already seen, it's a bit of a moving target)